etcd3/client.py

```diff
@@ -86,34 +74,94 @@ def __init__(self, alarm_type, member_id):
         self.member_id = member_id
 
 
+class Node(object):
+    """Represents an etcd cluster member."""
+
+    creds = None
+    time_retry = 300.0
+
+    def __init__(self, host, port, secure=True):
+        self.host = host
+        self.netloc = "{host}:{port}".format(host=host, port=port)
+        self.secure = secure
+        self.protocol = 'https' if secure else 'http'
```

Comment:
that's how etcd itself refers to its endpoints in the configuration. I checked and in fact there is still no official designation for the grpc protocol in terms of URI names. So I went with what coreos does.